var/spack/repos/builtin/packages/magma/package.py

```diff
@@ -23,8 +23,9 @@ class Magma(CMakePackage, CudaPackage, ROCmPackage):
     test_requires_compiler = True
 
     version('master', branch='master')
-    version('2.6.2rc1', commit='5959b8783e45f1809812ed96ae762f38ee701972')
-    version('2.6.1', sha256='6cd83808c6e8bc7a44028e05112b3ab4e579bcc73202ed14733f66661127e213', preferred=True)
+    version('2.6.2', sha256='f7892a14067d92f73714196e4d76e0354c8dc3da80053612ec98965ce6793627')
```

Comment:
```suggestion
    version('2.6.2', sha256='f7892a14067d92f73714196e4d76e0354c8dc3da80053612ec98965ce6793627', preferred=True)
```
`spack info magma` says 2.6.2rc1 is the preferred version because Spack sorts it as "newer" than 2.6.2. I think the concretizer will still choose 2.6.2 to avoid the deprecated version, but it's worth being explicit until this version is removed.